packages/api/src/schema/api-schema.yaml

```diff
@@ -1929,6 +1929,9 @@ components:
                   bitrate:
                     type: number
                     example: 449890
+            attestation:
+              $ref: "#/components/schemas/attestation"
```

Comment:
Hmm is this like a circular reference?

Reply:
No, it points correctly here: https://github.com/livepeer/studio/blob/42aa0e87e9869f7b29396efefa10ab22b6e6ab91/packages/api/src/schema/db-schema.yaml#L371

Reply:
Ohh I only realized now that this was a field and not a component schema 😅